subnet.json

```diff
@@ -1,3 +1,3 @@
 {
-    "proposerMinBlockDelay": 0
+    "proposerMinBlockDelay": 200000000
```

Comment:
what does this do?

Reply:
it's the block delay we set to 0 earlier. I was testing it out on local.